signup-front/src/pages/ApiEntreprise/demarches.json

```diff
@@ -81,7 +84,24 @@
         "entreprises_artisanales": true,
         "exercices": true,
         "attestations_fiscales": true,
-        "attestations_sociales": true
+        "attestations_sociales": true,
+        "actes_inpi": false,
```

Comment:
we cannot duplicate all the scopes for each demarche if we want this to be maintainable. Plz define only the modified scopes here.